docs.md

```diff
@@ -125,3 +125,12 @@ export default {
   },
 };
 ```
+
+To avoid Woodpecker workflow runs you should also consider adding an exclude condition. Otherwise the pull requests created and updates by this plugin triggers a workflow run itself. This could cause issues if your
```

Comment:
Adding such an filter would mean that linters checking formatting or spelling etc would be skipped. It might be better to have this as a hint if someone is searching for a way to skip those PRs.